### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- *Upgrade to version 1.0.3*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/framaforms_ynh%20PR39%20(yalh76)/)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "en": "Create online webforms and surveys",
     "fr": "Créez des formulaires et questionnaires en ligne"
   },
-  "version": "1.0.1+200715~ynh4",
+  "version": "1.0.3~ynh1",
```
Comment (yalh76) [inline manifest.json:9]: You should add an upgrade test from current version in https://github.com/YunoHost-Apps/framaforms_ynh/blob/testing/check_process

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — APPROVED** [review]
Just have to wait of a package_check test

**Comment by alexAubin** [comment]
!testme

**Comment by tituspijean** [comment]
Maybe all those warnings could be silenced? (cf. the test above)

**Review by yalh76 — APPROVED** [review]

**Pull request merged**